### PR TITLE
Send LogPostStatus after succesful log ingest

### PR DESCRIPTION
**Pull request by henridf**

### CODE REVIEW

File: zqd/api/api.go [hunk 1]
```diff
@@ -117,6 +117,13 @@ type PacketPostStatus struct {
 	MaxTime        *nano.Ts `json:"max_time,omitempty"`
 }
 
+type LogPostStatus struct {
+	Type    string   `json:"type"`
+	MinTime *nano.Ts `json:"min_time,omitempty"`
+	MaxTime *nano.Ts `json:"max_time,omitempty"`
+	Size    int64    `json:"size" unit:"bytes"`
+}
+
```
Comment (nwt) [inline zqd/api/api.go:126]: Nit: Put this after LogPostRequest.

File: zqd/handlers_test.go [hunk 2]
```diff
@@ -260,9 +260,20 @@ func TestPostLogs(t *testing.T) {
...
+			MinTime: &min,
+			MaxTime: &max,
+			Size:    80,
+		}
+		require.Equal(t, expected, status)
+
+		taskend := payloads[len(payloads)-1].(*api.TaskEnd)
+		assert.Equal(t, taskend.Type, "TaskEnd")
+		assert.Nil(t, taskend.Error)
+
```
Comment (nwt) [inline zqd/handlers_test.go:276]:
```suggestion
```

**Comment by mattnibs** [comment]
Can we hold off on merging for a sec? Just want to finish my review.

File: zqd/api/api.go (hunk 1 above)
Comment (henridf) [reply zqd/api/api.go:130]: done

**Comment by mattnibs** [comment]
Why have this response when the user could just issue a space/info request upon completion? I'm struggling to see a value add that justifies the added part.

**Comment by henridf** [comment]
Fair question @mattnibs, here's the missing PR description:

I added this out of symmetry with the pcap endpoint and with the expectation that it would evolve into periodic updates for large ingest jobs, in support of a progress bar. Similarly to how the pcap endpoint evolved.

**Comment by mattnibs** [comment]
@henridf thanks for the explanation. Sounds good to me then.

**Pull request merged**